_data/training-schools.yml

```diff
@@ -1,179 +1,177 @@
-- title: AEPSHEP (Asia-Europe-Pacific School of High-Energy Physics)
```

Comment:
Nothing happens in this file, it's just sorted by date and the dictionary keys are sorted alphabetically; all blanks are removed.